soccer/src/soccer/radio/packet_convert.hpp

```diff
@@ -36,19 +40,30 @@ void grsim_to_status(const Robot_Status& grsim, RobotStatus* status);
 
 void status_to_proto(const RobotStatus& status, Packet::RadioRx* proto);
 
+void status_to_ros(const RobotStatus& status, rj_msgs::msg::RobotStatus* msg);
+
+void ros_to_status(const rj_msgs::msg::RobotStatus& msg, RobotStatus* status);
+
 }  // namespace ConvertRx
 
 namespace ConvertTx {
 
 void to_rtp(const RobotIntent& intent, const MotionSetpoint& setpoint,
```

Comment:
I think the `to_*` functions are now unused? I haven't actually looked beyond this PR